### DISCUSSION TITLE
Is Clerk compatible with the Nextjs Turbopack Beta yet?

### DISCUSSION THREAD

**Discussion by valerius21** [clerk/javascript#1156] [Support request - DO NOT OPEN HERE]
Hi! I was wondering if the current version of `@clerk/nextjs` package (^4.17) supports the turbopack builder from the nextjs framework? I know it's in beta. I'm currently getting import errors, and I'm wondering if it's a "beta thing" or my fault. 😅

A screenshot is attached, if someone is interested in looking at it. I am using this template (https://github.com/shadcn/ui/tree/main/templates/next-template), and I was follwoing the latest documentation, as of today.

<img width="1446" alt="image" src="https://user-images.githubusercontent.com/10323945/236582716-5ede80b0-d466-4d83-8730-bf3a8410c3a8.png">

all packages are up-to-date.

**Comment by weleo** [comment]
Look like Clerk is not there yet. I'm receiving the following error when using `next 13.4.5` and `@clerk/nextjs 4.21.1`:

```
error during Next.js routing for /

Caused by:
- Error: Cannot find module '#crypto'

Debug info:
- Execution of get_from_source failed
- Execution of resolve_source_request failed
- Execution of NextRouterContentSource::get failed
- error during Next.js routing for /
- Error: Cannot find module '#crypto'
``````

**Reply by kiikoh** [reply]
Getting same thing still

**Comment by jescalan** [comment]
Still not yet - we're actively working on turbopack support, there will certainly be an announcement [on our changelog](https://clerk.com/blog) when we feel confident that we have it supported!
